translations/dutch/helptickets.ts

```diff
@@ -153,14 +153,14 @@ export const translations: Translations = {
-		"Please clearly explain why you should be unlocked and we will review it as soon as possible.": "",
-		"Please give us all relevant information on how you are connecting to Pokémon Showdown (if it is through mobile data, at home, a school or work network, etc), and we will review your case as soon as possible.": "",
-		"Please tell us which room you need assistance with and a global staff member will join your room as soon as possible.": "",
-		"If your issue pertains to battle mechanics or is a question about Pokémon Showdown, you can ask in the <<help>> chatroom.": "",
+		"Hi! All global staff members are busy right now and we apologize for the delay. ": "Hallo! Alle globale personeelsleden zijn momenteel druk bezig en we verontschuldigen ons voor de vertraging.",
+		"Please make sure you have given us the permission to check the PMs between you and the user you reported. You can also provide any relevant context; for example, a replay of a battle with the person you're reporting.": "Zorg er alsjeblieft voor dat je ons toestemming hebt gegeven om de PM's tussen jou en de gebruiker die je hebt gerapporteerd te controleren. Je kunt ook relevante context verstrekken; bijvoorbeeld een replay van een gevecht met de persoon die je rapporteert.",
+		'Please save the replay of the battle and provide a link to it in this chat, so we can see the harassment even if the battle expires. You can save the replay by clicking on the "Upload and share replay" button once the battle has ended.': 'Bewaar alsjeblieft de replay van het gevecht en geef een link naar deze in deze chat, zodat we de intimidatie kunnen zien, zelfs als het gevecht verloopt. Je kunt de replay opslaan door op de knop "Upload en deel replay" te klikken zodra het gevecht is afgelopen.',
+		"Make sure you have provided the correct username, and if its meaning or why it is offensive is not obvious, please explain why it should not be allowed.": "Zorg ervoor dat je de juiste gebruikersnaam hebt opgegeven, en als de betekenis ervan of waarom het ongepast is niet duidelijk is, leg dan alsjeblieft uit waarom het niet toegestaan zou moeten zijn.",
+		'Please save the replay of the battle and provide a link to it in this chat, so we can see the nicknames even if the battle expires. You can save the replay by clicking on the "Upload and share replay" button once the battle has ended.': 'Bewaar alsjeblieft de replay van het gevecht en geef een link naar deze in deze chat, zodat we de bijnamen kunnen zien, zelfs als het gevecht verloopt. Je kunt de replay opslaan door op de knop "Upload en deel replay" te klikken zodra het gevecht is afgelopen.',
+		"Please clearly explain why you should be unlocked and we will review it as soon as possible.": "Leg alsjeblieft duidelijk uit waarom je vrijgelaten zou moeten worden en we zullen het zo snel mogelijk beoordelen.",
```

Comment:
In some other translations we simply use "unlocked" instead of translating it, I think sticking to that is clearer